package.json

```diff
@@ -263,6 +265,10 @@
     }
   },
   "lint-staged": {
+    "public/locales/**/*.{json,po}": [
```

Comment:
Should we hold-off from adding this into `lint-staged` yet while figure out the whole process?

Reply:
Sure, that makes sense. Removed in 9c21c306f73376d811fe1f52e3579ef6d3c3360f